packages/web-components/src/components/content-block-horizontal/__stories__/README.stories.react.mdx

```diff
@@ -0,0 +1,67 @@
+import { Preview, Props, Description, Story } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import { PropTypesRef } from '@carbon/ibmdotcom-web-components/es/components-react/content-block/content-block.js';

```

Comment:
should this be importing prop types from `content-block-horizontal`?

Reply:
React wrappers seem to be unable to get the component proptypes if they are an extended from another one, but the Content Block Proptypes are able to be retrieved as normal